### ISSUE TITLE
Issues importing 1pif via Web

### ISSUE THREAD

**Issue by lodestone** [bitwarden/web#338]
## Import Issues

I just attempted to import my 1Password data into Bitwarden via the web.

Here's what happened:

+ I exported my 1Password (6.8.8 on macOS mojave) vault as 1pif format. 
+ Attempted to import the 1pif file via the web interface.
+ I got some errors about notes exceed 10,000 characters.
+ Found the offending large note entries and removed them.
+ Re-exported and attempted import via web again.
+ Watching the network requests and console I saw JSON parsing errors, which makes sense to me because when I opened that 1pif file, it did _NOT_ look like valid json.

***The file included an id (like `***5642bee8-a5ff-11dc-8314-0800200c9a66***`) after each entry:***

```json
{"uuid":"B7666370ACDC485DAA9D595891A5417D","updatedAt":1326733906,"securityLevel":"SL5","contentsHash":"6c737ac7","title":"Kids","secureContents":{},"txTimestamp":1504720833,"createdAt":1326733901,"typeName":"system.folder.Regular"}
***5642bee8-a5ff-11dc-8314-0800200c9a66***
{"uuid":"298574BB471A4B21B479BBE376563874","updatedAt":1491589969,"securityLevel":"SL5","contentsHash":"25279594","title":"OLD","secureContents":{},"txTimestamp":1504720833,"createdAt":0,"typeName":"system.folder.Regular"}

```

## Questions:

+ _Why is there a 10,000 character limit on the notes field of all things?_
+ _What's up with the id lines in the export file? Is that expected or not?_

**Comment by kspearrin** [comment]
1. We don't want users "abusing" the system with very large amounts of data stored on items. This is what file attachments are for.

2. This is just how 1Password formats their 1pif. It is expected

**Comment by lodestone** [comment]
> 1. We don't want users "abusing" the system with very large amounts of data stored on items. This is what file attachments are for.

That makes sense.

> 2. This is just how 1Password formats their 1pif. It is expected

If 1Password always formats, then why did the import fail claiming it was invalid JSON?

**Comment by kspearrin** [comment]
We don't parse it as valid JSON as is. We split the file on the ids and parse each JSON row as valid JSON.

Did you eventually get this all working?

**Comment by lodestone** [comment]
Yep, I did get it working. 

It would not accept the 1pif file with the weird ids. Either the console was raising errors about JSON invalidity or I was getting a similar error back from a network request -- I actually don't remember now which it was.

So I ended up opening the 1pif in vim and removing those ids and replacing them with a comma. 

Once I did that, it imported everything successfully as far as I can tell... 🤷‍♂️ 

I just wanted to make sure to report my experience in case this happens to **NOT** be anomalous.
